README.md

```diff
@@ -90,7 +90,7 @@ The default template URL of `https://github.com/openfaas/templates.git` can be o
 * 1st priority CLI input
 * 2nd priority `OPENFAAS_TEMPLATE_URL` environmental variable
 
-Help for all of the commands supported by the CLI can be found by running:
+Help for all the commands supported by the CLI can be found by running:
```

Comment:
This one seems a little ambiguous, perhaps a little bit too prescriptive? https://jakubmarian.com/is-it-all-the-or-all-of-the-in-english/